extensions/common/iam/verifiable-credentials/src/main/java/org/eclipse/edc/iam/verifiablecredentials/VerifiableCredentialValidationServiceImpl.java

```diff
@@ -75,11 +73,11 @@ private Result<Void> validateVerifiableCredentials(List<VerifiableCredential> cr
                 new HasValidSubjectSchema(mapper)));
 
         filters.addAll(additionalRules);
-        var results = credentials
+        return credentials
```

Comment:
better to simply add at the beginning of the method
```java
if(credentials.isEmpty()){
        return success();
}
```

this will be much more explicit, easier to read and avoids needless instantiation of classes